src/helics/core/BrokerBase.cpp

```diff
@@ -150,6 +151,24 @@ std::shared_ptr<helicsCLI11App> BrokerBase::generateBaseCLI()
         "--globaltime",
         globalTime,
         "specify that the broker should use a globalTime coordinator to coordinate a master clock time with all federates");
+    hApp->add_flag(
+        "--async",
+        asyncTime,
+        "specify that the federation should not synchronize time (time management is done outside of helics or not relevant");
```

Comment:
I think I'm having the same trouble with this description as the class description -- `AsyncTimeCoordinator` feels like it implies some different time coordination is still happening. It sounds like this option is really intended for mechanisms outside HELICS to provide time management, so maybe `ExternalTimeCoordinator` would match better with what it does and how it is used.